Billing.API.Test/InvoiceApiTest.cs

```diff
@@ -338,8 +338,7 @@ public async Task GetInvoices_WhenNoParameters_ReturnsNotFound()
         }
 
         [Theory]
-        [InlineData("accounts/doppler/1/invoice/invoice_2020-01-01_123.pdf")] //Deprecated route
-        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf")]
+        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf?signature=792naTFnk0doxkAi3G4Dt2ITSQttLcf6OypamgKuV0")]
```

Comment:
Hay posibilidades de hacerlo más corto?

```suggestion
        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf?s=792naTFnk0doxkAi3G4Dt2ITSQttLcf6OypamgKuV0")]
```

```suggestion
        [InlineData("accounts/doppler/1/invoices/invoice_2020-01-01_123.pdf?792naTFnk0doxkAi3G4Dt2ITSQttLcf6OypamgKuV0")]
```